main/at-spi2-core/APKBUILD

```diff
@@ -1,40 +1,34 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
-# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=at-spi2-core
-pkgver=2.28.0
-pkgrel=1
+pkgver=2.32.0
+pkgrel=0
 pkgdesc="Protocol definitions and daemon for D-Bus at-spi"
-url="http://www.linuxfoundation.org/en/AT-SPI_on_D-Bus"
+url="https://www.freedesktop.org/wiki/Accessibility/AT-SPI2/"
 arch="all"
 options="!check"  # Requires running dbus daemon.
 license="LGPL-2.0-or-later"
-depends=""
-depends_dev="libxtst-dev"
```

Comment:
I would think `-dev` should depend on `dbus-dev` at least.

Reply:
Huh, looking at abuild output it detects `pc:dbus-1>=1.5` just fine as dep